plugins/SEO-Tool/js/api/lighthouse.js

```diff
@@ -0,0 +1,54 @@
+function detailedLink(url, strategy) {
+	return 'https://googlechrome.github.io/lighthouse/viewer/?psiurl=' + url + '&strategy=' + strategy;
+}
+
+async function lighthouseApi(link, apiKey) {
+	const strategy = ['mobile', 'desktop'], pageData = {};
+
+	const proms = strategy.map(async (val, ind) => {
+		const api = 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed', parameters = {
+			url: encodeURIComponent(link),
+			key: apiKey,
+			category: ['performance', 'accessibility', 'best-practices', 'seo', 'pwa'],
```

Comment:
Since we are not using the pwa results, it'd be better to comment it out. Later as per the need we can uncomment it.